Dockerfile

```diff
@@ -1 +1,27 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java11:latest
+RUN useradd -r -g root tomcat
+
+ENV TOMCAT_VERSION=9.0.22  \
+    TOMCAT_MAJOR_VERSION=9
+
+ADD  https://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR_VERSION}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz /tmp
+
+RUN cd /tmp && \
+    tar -xf apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    rm apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    mv apache-tomcat-${TOMCAT_VERSION} /opt/tomcat/ && \
```

Comment:
How is it possible that the `/opt/tomcat` directory exists at this stage? I don't understand...

Reply:
@naveen-armedia  is this resolved?

Reply:
the destination directory or file are  created if not present when using `mv` command.